kotlin/src/test/kotlin/io/smallrye/mutiny/coroutines/MultiAsFlowTest.kt

```diff
@@ -147,13 +148,13 @@ class MultiAsFlowTest {
                 delay(100)
                 emit()
                 emit()
+                yield()
```

Comment:
this test got flaky, the onFailure was sometimes propagated before `flow.collect` retrieved the second item - forcing a context switch helped. need to observe this further on.